src/medCore/database/medDatabaseModel.cpp

```diff
@@ -224,8 +224,9 @@ QVariant medDatabaseModel::data(const QModelIndex& index, int role) const
 QVariant medDatabaseModel::headerData(int section, Qt::Orientation orientation, int role) const
 {
     if (orientation == Qt::Horizontal && role == Qt::DisplayRole)
-        return this->columnNames().at(section);
-
+    {
+        return QString("Database");
```

Comment:
It's the name of the tool, the database. The tool above is the filter of the database, and under it's the database. It was not named, and it's way clearer like that :)